extensions/Worker.Extensions.Storage.Queues/src/TypeConverters/QueueConverterBase.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Functions.Worker.Converters;
+using Microsoft.Azure.Functions.Worker.Core;
+using Microsoft.Azure.Functions.Worker.Storage.Queues;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    internal abstract class QueueConverterBase<T>  : IInputConverter
+    {
+        private readonly ILogger<QueueConverterBase<T>> _logger;
+
+        public QueueConverterBase(ILogger<QueueConverterBase<T>> logger)
```

Comment:
For base classes, I recommend taking in the plain `ILogger`. `ILogger<T>` is purely for the DI scenario - it has no use beyond that.

```suggestion
        public QueueConverterBase(ILogger logger)
```

but also it isn't used, so can be removed?

Reply:
Noted for future reference! But removing for now as unused